libvirt.go

```diff
@@ -316,20 +316,28 @@ func (d *Driver) Start() error {
 	// They wont start immediately
 	time.Sleep(5 * time.Second)
 
-	for i := 0; i < 90; i++ {
-		time.Sleep(time.Second)
+	for i := 0; i < 40; i++ {
 		ip, err := d.GetIP()
 		if err != nil {
 			return fmt.Errorf("%v: getting ip during machine start", err)
 		}
+
 		if ip != "" {
 			// Add a second to let things settle
 			time.Sleep(time.Second)
-			return nil
+			log.Infof("Found IP for machine: %s", ip)
+			d.IPAddress = ip
+			break
 		}
-		log.Debugf("Waiting for the VM to come up... %d", i)
+
+		log.Debugf("Waiting for machine to come up %d/%d", i, 40)
```

Comment:
This is starting to be very small details, but it would be better to define a const for the number of loops we are going to run, as if we want to change the number of iterations, we are bound to forget this one ;)